cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java

```diff
@@ -71,7 +72,8 @@
   private final Lock writeLock;
   private final KeyStore keyStore;
 
-  FileSecureStore(CConfiguration cConf) throws IOException {
+  @Inject
+  public FileSecureStore(CConfiguration cConf) {
```

Comment:
when you use ``Injector`` to get an instance in the guice module, you can turn this back to package-private